uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -510,3 +512,37 @@ function _tasksAssignments_allow_only_add_timeline_action (phpQueryObject $doc)
   $actionDropdown->after($addTimelineBtn);
   $actionDropdown->remove();
 }
+
+/**
+ * Remove columns from timeline
+ *
+ * @param phpQueryObject $doc
+ */
+function _tasksAssignments_remove_columns_from_timeline ($doc) {
+  $selectColumnHeader = $doc->find('table th:nth-child(6)');
+  $selectColumnBody = $doc->find('table td:nth-child(6)');
+  $referenceColumnHeader = $doc->find('table th:nth-child(4)');
+  $referenceColumnBody = $doc->find('table td:nth-child(4)');
+  $statusColumnHeader = $doc->find('table th:nth-child(3)');
+  $statusColumnBody = $doc->find('table td:nth-child(3)');
+
+  $elementsToRemove = [
+    $selectColumnHeader, $selectColumnBody,
+    $referenceColumnHeader, $referenceColumnBody,
+    $statusColumnHeader, $statusColumnBody
+  ];
```

Comment:
Please format in such a way that each item stays on its own line.

Reply:
Code is updated